docs/src/content/docs/it/reference/configuration.mdx

```diff
@@ -344,9 +345,85 @@ starlight({
+Consulta la guida ["Blocchi codice"](/it/guides/authoring-content/#blocchi-codice) per scoprire come utilizzare la sintassi Expressive Code nel tuo contenuto Markdown e MDX.
+
+Puoi utilizzare tutte le opzioni di configurazione standard di [Expressive Code](https://github.com/expressive-code/expressive-code/blob/main/packages/astro-expressive-code/README.md#configuration) nonché alcune proprietà specifiche di Starlight, impostandole nell'opzione `expressiveCode` di Starlight.
+Ad esempio, imposta l'opzione `styleOverrides` di Expressive Code per sovrascrivere il CSS predefinito. Ciò consente personalizzazioni come dare ai tuoi blocchi di codice angoli arrotondati:
+
+```js ins={2-4} 
+starlight({
+	expressiveCode: {
+		styleOverrides: { borderRadius: '0.5rem' },
+	},  
+});
+```
+
+Se vuoi disabilitare Expressive Code, imposta `expressiveCode: false` nella configurazione di Starlight: 
+
+```js ins={2}
+starlight({  
+	expressiveCode: false,
+});
+```
+
+In aggiunta alle opzioni standard di Expressive Code, puoi anche impostare le seguenti proprietà specifiche di Starlight nella tua configurazione `expressiveCode` per personalizzare ulteriormente il comportamento del tema per i tuoi blocchi di codice:
+
+#### `themes`
+
+**tipo:** `Array<string | ThemeObject | ExpressiveCodeTheme>`
+**predefinito:** `['starlight-dark', 'starlight-light']`  
+
+Imposta i temi utilizzati per dare stile ai blocchi di codice.
+Consulta la [documentazione `themes` di Expressive Code](https://github.com/expressive-code/expressive-code/blob/main/packages/astro-expressive-code/README.md#themes) per i dettagli dei formati di tema supportati.
+
+Per impostazione predefinita, Starlight utilizza le varianti chiara e scura del tema [Night Owl](https://github.com/sdras/night-owl-vscode-theme) di Sarah Drasner.
+
+Se fornisci almeno un tema chiaro e uno scuro, Starlight sincronizzerà automaticamente il tema del blocco di codice attivo con il tema corrente del sito.
+Configura questo comportamento con l'opzione [`useStarlightDarkModeSwitch`](#usestarlightdarkmodeswitch).
+
+#### `useStarlightDarkModeSwitch`  
+
+**tipo:** `boolean`
+**predefinito:** `true`  
+
+Quando è impostato su `true`, i blocchi di codice passano automaticamente tra temi chiari e scuri quando cambia il tema del sito.  
+Quando è impostato su `false`, devi aggiungere manualmente CSS per gestire il passaggio tra più temi.  
+
+:::note
+Quando imposti `themes`, devi fornire almeno un tema chiaro e uno scuro affinché il cambio di tema in modalità scura di Starlight funzioni.
+:::
+
+#### `useStarlightUiThemeColors`  
+
+**tipo:** `boolean`
+**predefinito:** `true` se `themes` non è impostato, altrimenti `false`   
+
+Quando è impostato su `true`, le variabili CSS di Starlight vengono utilizzate per i colori degli elementi dell'interfaccia utente dei blocchi di codice (sfondi, pulsanti, ombre ecc.), in linea con il [tema di colore del sito](/guides/css-and-tailwind/#theming).  
```

Comment:
Qua manca `/it/` nel link e il frammento ache in italiano `#temi` per corrispondere con la intestazione tradotta:

```suggestion
Quando è impostato su `true`, le variabili CSS di Starlight vengono utilizzate per i colori degli elementi dell'interfaccia utente dei blocchi di codice (sfondi, pulsanti, ombre ecc.), in linea con il [tema di colore del sito](/it/guides/css-and-tailwind/#temi).  
```